core/src/main/java/org/springframework/security/access/expression/SecurityExpressionHandler.java

```diff
@@ -42,7 +43,10 @@
 	 * Provides an evaluation context in which to evaluate security expressions for the
 	 * invocation type.
 	 */
-	EvaluationContext createEvaluationContext(Authentication authentication, T invocation);
+	default EvaluationContext createEvaluationContext(Authentication authentication, T invocation) {
```

Comment:
Is there a reason to default this method in the same commit?

Reply:
Shoot, good catch! It's left over from an experiment. I'll revert.